server/game/gameSystems/GameSystemLibrary.ts

```diff
@@ -114,9 +115,9 @@ export function deploy<TContext extends AbilityContext = AbilityContext>(propert
 export function defeat<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<IDefeatCardProperties, TContext> = {}): CardTargetSystem<TContext> {
     return new DefeatCardSystem<TContext>(propertyFactory);
 }
-// export function discardCard(propertyFactory: PropsFactory<DiscardCardProperties> = {}): CardGameAction {
-//     return new DiscardCardAction(propertyFactory);
-// }
+export function discardCard<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<IDiscardProperties, TContext> = {}): DiscardCardSystem<TContext> {
```

Comment:
Since we're going to end up with a plethora of discard systems we should probably be super specific with the naming. Maybe in the vein of `drawSpecificCard` we make this `discardSpecificCard`? And same for the class itself. Lmk what you think

Reply:
Yup done